### PR TITLE
send reaction events to server

### PR DESCRIPTION
**Pull request by elimelt**
need to add this to env:

```sh
export INTERNSHIP_CHANNEL_ID=1298121534489231421
export NEW_GRAD_CHANNEL_ID=1298149664708890676
```

### CODE REVIEW

File: APIs/SweccAPI.py [hunk 1]
```diff
@@ -60,4 +64,31 @@ async def reset_password(self, discord_username, id):
...
+        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
+
+    async def process_reaction_event(self, payload, type):
+        user_id, channel_id, emoji = (
+            payload.user_id,
+            payload.channel_id,
+            payload.emoji,
+        )
+
+        if channel_id in self.reaction_channel_subscriptions and emoji.name == "✅":
```
Comment (Advayp) [inline APIs/SweccAPI.py:76]: Maybe store this emoji as a constant somewhere in the class?

File: APIs/SweccAPI.py [hunk 2]
```diff
@@ -60,4 +64,31 @@ async def reset_password(self, discord_username, id):
 
         response = requests.post(f"{self.url}/members/reset-password/", headers=self.headers, json=data)
         data = response.json()
-        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
\ No newline at end of file
+        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
```
Comment (Advayp) [inline APIs/SweccAPI.py:67]: Why are we returning this particular value here (the password reset confirm thing)?

Comment (elimelt) [reply APIs/SweccAPI.py:67]: You're probably not seeing the rest of the `reset_password` function. Expand diff on the left and it'll make sense

File: APIs/SweccAPI.py (hunk 1 above)
Comment (elimelt) [reply APIs/SweccAPI.py]: done

File: APIs/SweccAPI.py (hunk 2 above)
Comment (elimelt) [reply APIs/SweccAPI.py:68]: if not though lmk

Comment (Advayp) [reply APIs/SweccAPI.py:68]: Oh you're right, I see it now

**Review by Advayp — APPROVED** [review]
LGTM

**Pull request merged**